React/main/src/app/components/Board/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import Square from '~components/Square';
+
+import style from './styles.scss';
+
+class Board extends React.Component {
```

Comment:
We define a component class like this:

class Board extends Component

and import component like this: 

import React, { Component } from 'react';